### PR TITLE
Updated prerequisites to include Ruby and Sass with install directions

### PR DESCRIPTION
**Pull request by tatethurston**

### CODE REVIEW

**Comment by codydaig** [comment]
LGTM 
@tatethurston Welcome and thanks! :-D

File: README.md [hunk 1]
```diff
@@ -31,6 +32,12 @@ $ npm install -g bower
 $ npm install -g grunt-cli
 ```
 
+* Sass - You're going to use Sass[Sass](http://sass-lang.com/) to compile CSS durring your grunt task. Make sure you have ruby and then install Sass using gem install:
```
Comment (codydaig) [inline README.md]: @tatethurston Sass[Sass] shows up as a duplicate. Remove the Sass from before the [].

**Comment by codydaig** [comment]
@tatethurston I left you a line comment on a type I just discovered in this. Can you fix that before we merge this in?

**Comment by tatethurston** [comment]
@codydaig All fixed.

**Comment by lirantal** [comment]
LGTM

**Comment by codydaig** [comment]
@tatethurston Squash, and you'll be good to go. Also your git config isn't quite set-up correctly. May want to adjust that if you want credit for the commit. :)

File: README.md [hunk 2]
```diff
@@ -31,6 +32,12 @@ $ npm install -g bower
 $ npm install -g grunt-cli
 ```
 
+* Sass - You're going to use [Sass](http://sass-lang.com/) to compile CSS durring your grunt task. Make sure you have ruby and then install Sass using gem install:
```
Comment (ilanbiala) [inline README.md]: typo on during

**Comment by tatethurston** [comment]
@codydaig Squashed and fixed the typo mentioned by @ilanbiala. Regarding my git config, what do you see that is set incorrectly?

**Comment by tatethurston** [comment]
resubmitted PR: https://github.com/meanjs/mean/pull/1000

**Pull request closed without merging**